examples/DotNet/Program.cs

```diff
@@ -1,9 +1,3 @@
-using Microsoft.AspNetCore.Authentication;

```

Comment:
Some of these using were not require for the specific sample (Microsoft.AspNetCore.Authentication was not needed since no authentication was confirmed).

For others, like `Microsoft.AspNetCore.Builder` we're taking advantage of ASP.NET Core's implicit usings feature to trim down the number of imports we're using.